### PR TITLE
Add support for Apple Swift language

### PR DESCRIPTION
**Pull request by ankitpati**
[Apple Swift](https://swift.org/) is a new, but established language, judging by the [activity](https://github.com/apple/swift/pulse) on its [GitHub repository](https://github.com/apple/swift). I believe that it deserves to be supported by Geany. Kindly consider this patch for inclusion.

### CODE REVIEW

File: data/filedefs/filetypes.Swift.conf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# multiline comments
+comment_open=/*
+comment_close=*/
+
+comment_use_indent=true
+
+[build-menu]
+FT_00_LB=chmod +x
+FT_00_CM=chmod +x "%f"
+FT_00_WD=
```
Comment (b4n) [inline data/filedefs/filetypes.Swift.conf]: that's odd.  I suppose it's for the "Execute as Script" below?  I think it's better to put the interpreter on that other command and not add `chmod +x`, which is probably not very convenient, and not really sounds like a really Swift-related thing.

**Comment by b4n** [comment]
Ah, and it misses an update to *data/Makefile.am*.

File: data/filedefs/filetypes.Swift.conf (hunk 1 above)
Comment (ankitpati) [inline data/filedefs/filetypes.Swift.conf]: On Unix-like systems, Swift files can be executed as scripts using shebangs, as shown in the code below. That requires the execute bit to be set.

```
#!/usr/bin/env swift

import Glibc
import Foundation

print("hello, world")
```

Comment (b4n) [inline data/filedefs/filetypes.Swift.conf]: That should be the same as running `swift "%f"`, with or without execution bit.  The execution bit just allows the OS to execute, see the shebang and forward execution to `/usr/bin/env swift "$@"`

**Comment by ankitpati** [comment]
Making a new PR to avoid making two commits for one change, as outlined in the HACKING file.

**Comment by b4n** [comment]
You or us can always merge commits after the fact, don't worry about that when making the initial changes.

**Comment by ankitpati** [comment]
Please pardon my inexperience with GitHub pull requests. PR #1323 incorporates the changes requested.

**Comment by b4n** [comment]
No problem, I just wanted to spare you the extra work, a new PR is fine too

**Pull request closed without merging**